backend/support/api/admin_api_v1_1_1/create_views.sql

```diff
@@ -0,0 +1,92 @@
+
+begin;
+
+
+---------------------------------------
+-- accesses
+---------------------------------------
+-- public.audit_access definition
+
+-- Drop table
+
+-- DROP TABLE public.audit_access;
+
+CREATE OR REPLACE VIEW admin_api_v1_1_0.audit_access AS
```

Comment:
There are a few in this file:

```suggestion
CREATE OR REPLACE VIEW admin_api_v1_1_1.audit_access AS
```